### DISCUSSION TITLE
Native connector to Confluence

### DISCUSSION THREAD

**Discussion by ruschestor** [open-webui/open-webui#5877] [Ideas]
Hello.
Confluence is one of the most popular intranet workspaces, often with a huge amount of knowledge.
At the moment Web Search can't (?) authorize on websites/portals.
The Danswer (https://github.com/danswer-ai/danswer) has this feature - https://github.com/danswer-ai/danswer/tree/main/backend/danswer/connectors/confluence
I would appretiate this could be implemented in the near future.
Thank you in advance!

**Comment by tjbck** [comment]
https://github.com/open-webui/open-webui/issues/5872

**Comment by longfei-zhang** [comment]
@tjbck The above link is 404 now. Where can I find the latest info?

**Comment by juan-abia** [comment]
I don't think it makes sense to continue creating more in house integrations. All integrations like this should be done with Model Context Protocol. Let's not make the code dinosaur bigger 🙏

**Reply by longfei-zhang** [reply]
Do you know if there are any existing implementations of this requirement using the Model Context Protocol?

**Comment by RomainNeup** [comment]
If anyone is looking for a tool for it, I built 2 tools : [confluence search](https://openwebui.com/t/romainneup/confluence_search) and [confluence get page by id](https://openwebui.com/t/romainneup/confluence_page).
Source code is available here : https://github.com/RomainNeup/open-webui-utilities
Hope it will fits your needs (and if not don't hesitate to open an issue on Github)

**Reply by ruschestor** [reply]
> If anyone is looking for a tool for it, I built 2 tools : [confluence search](https://openwebui.com/t/romainneup/confluence_search) and [confluence get page by id](https://openwebui.com/t/romainneup/confluence_page). Source code is available here : https://github.com/RomainNeup/open-webui-utilities Hope it will fits your needs (and if not don't hesitate to open an issue on Github)

@RomainNeup, thanks! Could you please clarify on which editions (Cloud, Data Center) and versions (6.x, 7.x, 8.x, 9.x) did you test the tool?
